Gemfile.lock

```diff
@@ -166,4 +166,4 @@ RUBY VERSION
    ruby 2.6.5p114
 
 BUNDLED WITH
-   1.17.3
+   2.2.20
```

Comment:
**question:** was this meant to be `2.20.0`? Does that explain why the build failed?

Reply:
I think so -- probably the same issue that the build for [the one in `identity-idp-config`](https://github.com/18F/identity-idp-config/pull/440) was running into?

Reply:
So the version number you set it to is correct (the title is probably a typo), but I actually think it's because the `ruby-2.6.5` Docker image doesn't include the correct bundler version (which was the issue in `identity-idp-config`.